x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterQueryWithPartialResultsIT.java

```diff
@@ -346,6 +351,42 @@ public void testFailSearchShardsOnLocalCluster() throws Exception {
         }
     }
 
+    public void testResolutionFailures() throws Exception {
```

Comment:
Check out also https://github.com/elastic/elasticsearch/pull/129957/ - I've made some tests for things that weren't handled properly.